### DISCUSSION TITLE
Nginx Proxy Manager how to use

### DISCUSSION THREAD

**Discussion by the-grosser** [NginxProxyManager/nginx-proxy-manager#3688] [General]
Hello
sorry, but my English is not so good, but I'll try to explain or describe my problem anyway.
I have a general question about using the Nginx Prox Manager. Namely the following.

Is it necessary to run the Nginx Proxy Manager on the same server, in the same network, to run different web services (e.g. Nextcloud, Zammad etc.)?

Many thanks for the effort and best regards

the-grosser

**Comment by Vaadasch** [comment]
Hi, 
That is unclear.
No you don't need to have NPM on the same server.
Given the proper settings, you can proxy Google if you want from your server.
The only limitation is that NPM need to have access to the webservice you want it to serve.
Be sure that your link between your NPM and you webservices are secure. 
It is easy on a single server but depend on your structure.
